src/Data/Integer/DivMod.agda

```diff
@@ -13,7 +13,7 @@ open import Data.Integer.Properties
 open import Data.Nat.Base as ℕ using (ℕ; z≤n; s≤s; z<s; s<s)
 import Data.Nat.Properties as ℕ
 import Data.Nat.DivMod as ℕ
-import Data.Sign as S
+import Data.Sign.Base as S
```

Comment:
Should be `as Sign`? #2280

Reply:
Even better: delete altogether as it's not used!